lib/addresses.js

```diff
@@ -75,8 +77,8 @@ AddressController.prototype.getAddressSummary = function(address, options, callb
       totalSentSat: summary.totalSpent,
       unconfirmedBalance: summary.unconfirmedBalance / 1e8,
       unconfirmedBalanceSat: summary.unconfirmedBalance,
-      unconfirmedTxApperances: summary.unconfirmedAppearances, // misspelling - ew
```

Comment:
Note: this will break the API which has been expecting this mis-spelling for years (since Insight came out). So we'll probably want to make sure tools and UIs which consume this are aware of the change, or phase it out by putting both the mis-spelled version and the correct version in for a while, w/some kind of deprecation notice for the former.

Reply:
I knew you would say that :) I added it the misspellings back and put a deprecation notice in lieu of the yuck/ew. Like this we can phase it out slowly.